setup.py

```diff
@@ -14,7 +14,7 @@
     package_data={'pdfminer': ['cmap/*.pickle.gz']},
     install_requires=[
         'chardet ; python_version > "3.0"',
-        'pycryptodome',
+        'cryptography',
```

Comment:
We could also move cryptography to extra_requires. Then most people wont install cryptography at all. For pycryptodome that made sense, since it has these dependencies on c.

But for cryptography its much less relevant. It only has a few dependencies:

```
cryptography==2.9.2
  - cffi [required: >=1.8,!=1.11.3, installed: 1.14.0]
    - pycparser [required: Any, installed: 2.20]
  - six [required: >=1.4.1, installed: 1.14.0]
```

Reply:
So you do or do not want it moved to `extra_requires`?

Reply:
The issue/request I opened is agnostic about methods of reducing install size. What matters is to get it down for Alpine Linux. I'll see whether I can test it quickly on my docker build.

Reply:
Ok, I'm not good enough with Docker to build the work in progress within Docker and test for size. If you have a neat package I can simply download and install from within I could do the test, though. Otherwise, I guess no harm in implementing it first without `extra_requires` so as to be non-breaking, and then do that in a second step.
I just learned I can also simply uninstall the build tools I had to install again to make the image leaner.

Reply:
I made a comment on the referenced issue about cryptography which builds a wheel for alpine linux.  You should be able to verify for yourself that pdfminer + cryptography will work by simply installing cryptography.

Reply:
Fantastic :) That sounds like switching to cryptography should address the reason for creating the issue.

Reply:
Leave it in install_requires. Using extra_requires was a sensible idea if we were stuck with pycryptodome.